Basic/DynamicAddressablesNetworkPrefabs/README.md

```diff
@@ -1,27 +1,82 @@
+  <br><br>
+
+- Scene 05_API Playground Showcasing All Post-Connection Use-Cases
+  - This scene serves as an API playground to test how all of the use-cases in can work tandem.
+<br><br><br>
+
+## Known Limitations
+- If you have NetworkConfig.ForceSamePrefabs enabled, you can only modify your prefab lists **before** starting
   networking, and the server and all connected clients must all have the same exact set of prefabs
-  added via this method before connecting
-- Adding a prefab on the server does not automatically add it on the client - it's up to you
+  added via this method before connecting.
+
+- Adding a prefab on the server **does not** automatically add it on the client - it's up to you
   to make sure the client and server are synchronized via whatever method makes sense for your game
-  (RPCs, configs, deterministic loading, etc)
-- If the server sends a Spawn message to a client that has not yet added a prefab for, the spawn message
-  and any other relevant messages will be held for a configurable time (default 1 second, configured via
-  NetworkConfig.SpawnTimeout) before an error is logged. This is intented to enable the SDK to gracefully
-  handle unexpected conditions (slow disks, slow network, etc) that slow down asset loading. This timeout
-  should not be relied on and code shouldn't be written around it - your code should be written so that
+  (RPCs, configs, deterministic loading, etc). This sample lists some of those possible methods with basic implementations you can adapt to your various use cases.
+
+- If the server sends a Spawn message to a client that does not yet have the corresponding prefab loaded, the spawn message
+  (and any other relevant messages) will be held for a configurable time before an error is logged (default 1 second, configured via
+  NetworkConfig.SpawnTimeout). This is intented to enable the SDK to gracefully
+  handle unexpected conditions that slow down asset loading (slow disks, slow network, etc). This timeout
+  should not be relied upon and code shouldn't be written around it - your code should be written so that
   the asset is expected to be loaded before it's needed.
-- Currently it's impossible to latejoin for clients after a dynamic prefab has been spawned by the server. The reason for this is that the initial sync doesn't allow us any time to load prefabs that are yet absent on the client.
+  
+- It's currently impossible for clients to late join after a dynamic prefab has been spawned by the server - this is because the initial sync doesn't allow us any time to load prefabs that are aren't yet loaded on the client.
```

Comment:
@fernando-cortez can you check the closed github comments I sent you in chat, I already mentioned this should be removed.